modules/alloy/src/test/ts/browser/position/SelectionInFramePositionTest.ts

```diff
@@ -72,6 +76,33 @@ UnitTest.asynctest('SelectionInFramePositionTest', (success, failure) => {
       });
     };
 
+    const cAssertBelowSelection = NamedChain.bundle((data: any) => {
+      const frame = data.classic.element;
+      const root = SugarElement.fromDom(frame.dom.contentWindow.document.body);
+      const popup = data.popup;
+      const path = data.path3;
+      const range = toDomRange(Cursors.calculate(root, path));
+
+      const frameRect = frame.dom.getBoundingClientRect();
+      const selectionBox = range.getBoundingClientRect();
+      const popupBox = popup.element.dom.getBoundingClientRect();
+      assert.isAtLeast(popupBox.top, selectionBox.bottom + frameRect.top);
+      assert.approximately(popupBox.top, selectionBox.bottom + frameRect.top, 5);
+      return Result.value(data);
+    });
+
+    const cScrollIntoView = (rangeLabel: string, scrollLabel: string, errorMessage: string): Chain<NamedData, NamedData> => {
```

Comment:
```suggestion
    const cScrollIntoView = (rangeLabel: string, scrollLabel: string, errorMessage: string): NamedChain => {
```
This can just be this which saves declaring `NamedData` which we should be avoiding.